question25/register.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link href="style.css" rel="stylesheet" />
+        <script src="main.js" defer></script>
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <div id="js-wrapper" class="wrapper">
+            <h1>会員登録</h1>
+            <form method="post" id="form" novalidate>
+                <div>
+                    <ul>
+                        <li>
+                            <label for="name">ユーザー名</label>
+                            <input type="text" id="name" name="user_name" required data-maxlength="16" class="input"/>
+                            <sapn class="error"></span>
+                        </li>
+                        <li>
+                            <label for="mail">メールアドレス</label>
+                            <input type="email" id="mail" name="user_email" required class="input"/>
+                            <sapn class="error"></span>
+                        </li>
+                        <li>
+                            <label for="password">パスワード</label>
+                            <input type="password" id="password" name="user_password" required class="input"/>
+                            <sapn class="error"></span>
+                        </li>
```

Comment:
```suggestion
                            <span class="error"></span>
                        </li>
```

Reply:
Oh no...! Thank you for pointing it out! 
Fixed commit : https://github.com/csakyo/homework/pull/39/commits/497fe18b36fdcc9b324da81ff9095ddac5ece648